Pattern.java

```diff
@@ -0,0 +1,5 @@
+public class Pattern {
+    public static void main(String[] args) {
+        System.out.println("Pattern Printing...");
```

Comment:
Could you ignore using hard coded string? It would be better to use a constant.

Reply:
Can't you do it from your side, i think you are too experiance and will make this small changes right ?

Reply:
Sure

Reply:
Looks good!